### ISSUE TITLE
How to unset the tcsh ignoreeof var using the command option

### ISSUE THREAD

**Issue by kaushalmodi** [asciinema/asciinema#154]
I have the below record option in my `~/.config/asciinema/config` file:

```
[record]
command = /bin/tcsh
```

I would like to unset the tcsh special variable `ignoreeof` only in this tcsh session started by asciinema so that I can hit `Ctrl + D` to quickly stop the recording (instead of typing `exit`). Is there a way to do that?

I do not have the `ignoreeof` unset by default so that I don't accidently kill my tcsh sessions.

**Comment by sickill** [comment]
Can this option be passed to tcsh as a command line arg maybe?

**Comment by kaushalmodi** [comment]
I had looked up the tcsh man page and tried doing 

```
command = /bin/tcsh -c 'unset ignoreeof'
```
But of course that did not work. I didn't exactly understand from the man pages on how to use the `-c` arg. Hopefully someone here using tcsh can help me. 

I also tried playing with different quotes, without quotes, other tcsh args that I thought would work. But nothing helped. 

It would be great to know how and if I can do something like `/bin/tcsh` and then `unset ignoreeof` within that new shell process (and then initiate the recording), using the asciinema `command` option.
